transfers/pre-transfer/fix_unlv_checksums.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python
+
+from __future__ import print_function
+
+import sys
+import os
+
+
+HASHFUNC_EXT = 'md5'
+
+
+def find_objects(path, ignore_dirs=None):
```

Comment:
Yes I am! I thought this was nicer because I'm decoupling the process of listing the files in the way I want from its consumer. Does that make sense? I'm happy to use `os.walk` if that's the practice.